ouroboros-network-protocols/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Non-Breaking changes
 
+* Added `AnchoredFragment.splitAtSlot`
```

Comment:
This should go to the `ouroboros-network-api` changelog, right?

Reply:
Fixed.